From 7dd4d4c5590ac01f9a05df57b0b5bba4480e6793 Mon Sep 17 00:00:00 2001
From: Saahil Tomar <quic_saahtoma@quicinc.com>
Date: Wed, 30 Aug 2023 15:32:11 +0530
Subject: [PATCH] arm64: dts: qcom: ipq9574: Update Makefile for RDP475 &
 RDP476 dts.

This RDP is based on AL02-C6, with changes in
ethernet for GPON enablement

Change-Id: Ife94351bf1c4eb4f98ee0875981f570e393254a8
Signed-off-by: Saahil Tomar <quic_saahtoma@quicinc.com>
---
 arch/arm/boot/dts/Makefile        | 2 ++
 arch/arm64/boot/dts/qcom/Makefile | 2 ++
 2 files changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/Makefile b/arch/arm/boot/dts/Makefile
index 614b3c3510ee..46106ac7b503 100644
--- a/arch/arm/boot/dts/Makefile
+++ b/arch/arm/boot/dts/Makefile
@@ -1065,6 +1065,8 @@ dtb-$(CONFIG_ARCH_QCOM) += \
 	ipq9574-rdp461.dtb \
 	ipq9574-rdp467.dtb \
 	ipq9574-rdp469.dtb \
+	ipq9574-rdp475.dtb \
+	ipq9574-rdp476.dtb \
 	ipq9574-db-al01-c1.dtb \
 	ipq9574-db-al01-c2.dtb \
 	ipq9574-db-al01-c3.dtb \
diff --git a/arch/arm64/boot/dts/qcom/Makefile b/arch/arm64/boot/dts/qcom/Makefile
index 82773f6e2e58..31c62cce1ffa 100644
--- a/arch/arm64/boot/dts/qcom/Makefile
+++ b/arch/arm64/boot/dts/qcom/Makefile
@@ -36,6 +36,8 @@ dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-rdp459.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-rdp461.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-rdp467.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-rdp469.dtb
+dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-rdp475.dtb
+dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-rdp476.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-db-al01-c1.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-db-al01-c2.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-db-al01-c3.dtb
-- 
2.34.1

